src/hivpy/population.py

```diff
@@ -36,21 +39,34 @@ def _sample_parameters(self):
             'avg_max_age': avg_max_age,
         }
 
+    def _prob_HIV_initial(self, age):
+        """Completely arbitrary placeholder function for initial HIV presence in population"""
+        return np.clip(self.hiv_a*age**2 + self.hiv_b*age + self.hiv_c, 0.0, 1.0)
+
+    def _initial_HIV_status(self):
+        """Initialise HIV status based on age (& sex?)"""
+        """Assume zero prevalence for age < 15"""
+        hiv_status = self._prob_HIV_initial(self.data["age"]) > np.random.rand(self.size)
+        return hiv_status
```

Comment:
In an ideal world (that we don't live in!), all such initialisation functions (and the associated params) would be in an appropriate specific "module" and not in the Population class. I don't know if this would fit better in demographics or a separate hiv module - depends on whether we include more details of transmission dynamics, perhaps.